packages/manager/modules/office/src/dashboard/office/license/user/user.routing.js

```diff
@@ -0,0 +1,10 @@
+import template from './USER.html';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('office.product.user', {
+    url: '',
```

Comment:
isn't the url /user ?

Reply:
Yes as mentionned here also: https://github.com/ovh/manager/pull/3412#discussion_r468370085, The user tab is the landing tab. I based myself on pci architecture (which is the most recent) to structure tabs and url. Here the `user` template should be included into the parent view but it requires more refactoring with merging templates and controllers as it is the landing tab so we have something corresponding to the /license/office/{licenseId}. Please let me know if you think otherwise

Reply:
ok it's fine i was just wondering if it was intentional